packages/core/README.md

```diff
@@ -95,3 +95,133 @@ return choices.map(getValuePair).map((choice) => (
     </option>
 ));
 ```
+
+### Vind
+
+Vind er et subset av [Tailwind CSS](https://tailwindcss.com), med grunnlag i Jøkuls design tokens, som fokuserer på:
```

Comment:
Forsøker meg på et subset av Tailwind, som vi pratet om på et nylig frontendforum.

Jeg tenkte i utgangspunktet gjøre dette opt-in, og at man må ha med `core` for å få satt sånt som fontfamilie og CSS reset. Er åpen for diskusjon her.

- Vil vi ha dette som default?
- Skal `vind`, hvis den holdes adskilt, gjøre seg uavhengig av `core.css` (og unngå at verbose utilklasser o.l. lastes ned)?

Reply:
En liten fordel(?) om det er en del av `core.css` er at vi kan bruke det i komponenter.

Reply:
Tanken har slått meg også. Men da legger vi opp til at andre team "må" installere og sette opp Tailwind eller noe tilsvarende. Vil vi egentlig det?

Reply:
En annen tanke er om vi skal droppe util-klasser fullstendig fra Jøkul. Om man vil ha det, så brukes det gjennom Tailwind eller tilsvarende basert på tokens-fila. Om man ikke vil ha det, så har ikke Jøkul noen meninger om det?

Reply:
Det er lettere å legge til ting enn å fjerne dem, så hvis vi er i tvil heller jeg mot å droppe `vind.css` i denne omgang.

Reply:
Imo er det litt enten eller. Det kan hende jeg er mer tilbøyelig for å dra inn tailwind/uno enn de fleste, så jeg føler at noen andre bør si noe her også.

Reply:
Personlig har eg ingenting imot litt Vind i segla når eg koder (beklager, kunne ikkje dy meg).

Det kan hende det hadde vore like greit å innstallere tailwind og sette opp config, men når kjem eg til å gjøre det?

Vind er bra. Vind utanfor core trur eg er best:
- Alle som vil kan få util-klassar som er i tråd med designsystemet effortlessely (men dei som ikkje vil, skal slippe)
- Dei som ikkje bruker util-klassane slipper å lære dei for å kunne endre på jøkul-komponentane